Source/DafnyCore/Verifier/BoogieGenerator.cs

```diff
@@ -18,6 +18,7 @@
 using Microsoft.Boogie;
 using static Microsoft.Dafny.Util;
 using DafnyCore.Verifier;
+using JetBrains.Annotations;
```

Comment:
Is this used, or was it added by the IDE when auto-completing something you didn't end up using? I ask because I've had that happen quite a few times at this point. :)

Reply:
It looks like it's used for a warning suppression elsewhere, but neither are necessary and I think it would be better to forgo IDE-specific imports. I'll remove this in a patch.

Reply:
On second thought, I see that these annotations are used elsewhere in the project (and explicitly included in the `.csproj` file), so I'll keep these changes as they help the IDE catch potential issues.